### PR TITLE
docs: add snyder as sponsor to the doc website

### PR DESCRIPTION
**Pull request by Laphatize**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] vitest-angular
- [ ] astro-angular
- [ ] create-analog
- [ ] router
- [ ] platform
- [X] content
- [ ] nx-plugin
- [ ] trpc

## What is the current behavior?
There is no Snyder logo on the sponsors page. (https://analogjs.org/)

## What is the new behavior?
I added the Snyder logo to the page.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by brandonroberts** [comment]
Thanks @Laphatize. Will you run prettier on the files for cleanup?

**Comment by Laphatize** [comment]
Should be good to go now.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "node": "^18.13.0 || ^20.0.0",
     "pnpm": "^8.0.0"
   },
-  "packageManager": "pnpm@8.9.0",
+  "packageManager": "pnpm@8.9.0+sha512.ef8859938e1f0537b9fcf0309044311392f3b38b344176e6cd4fdee21b225524b0ac2a1b08ae2ae2ddd5b3ff4b8ca496d6778e950f3e7c9f555b81325982441c",
```
Comment (brandonroberts) [inline package.json:27]: Revert this change as its unrelated
```suggestion
  "packageManager": "pnpm@8.9.0",
```

**Review by brandonroberts — APPROVED** [review]

**Comment by benpsnyder** [comment]
Please don't merge yet. I am having @Laphatize make changes to how the SVGs are loaded in under static.

**Comment by benpsnyder** [comment]
Good to go -- thx

**Comment by Laphatize** [comment]
Should be good to go now.

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @Laphatize for doc
